packages/client/src/store.tsx

```diff
@@ -62,6 +62,7 @@ getMutableState(DomainConfigState).merge({
 
 export default function ({ children }): JSX.Element {
   const { t } = useTranslation()
+  const isLocation = window.location.pathname.includes('/location')
```

Comment:
let's pass this in rather than asume it from the pathname. we may want this functionality in other routes in the future